pkg/generate/code/set_sdk.go

```diff
@@ -784,8 +784,8 @@ func setSDKReadMany(
 			}
 		}
 
-		// Field renames are handled in getSanitizedMemberPath
-		resVarPath, err = getSanitizedMemberPath(memberName, r, op, sourceVarName)
+		// Field renames are handled in GetSanitizedMemberPath
+		resVarPath, err = r.GetSanitizedMemberPath(memberName, op, sourceVarName)
 		if err != nil {
```

Comment:
Just double checking: Why do we not just do `if err == nil { continue }` here, instead?

Reply:
In this context, `err == nil` means there was a field in the ReadMany shape that matched a field in the CRD exactly. 

For example, [DescribeReplicationGroupsInput](https://docs.aws.amazon.com/sdk-for-go/api/service/elasticache/#DescribeReplicationGroupsInput) is a ReadMany call, but has a *ReplicationGroupId* (note singular) field.